src/repl/repl_core.ts

```diff
@@ -142,15 +143,14 @@ export function runRepl(
     console.log();
 
     // Start up the REPL.
-    const repl: IRepl = {
-        server: replServer.start({
-            prompt: '% ',
-            input: process.stdin,
-            output: process.stdout,
-            eval: processReplInput,
-            writer: myWriter,
-        }),
-    };
+    const server = replServer.start({
+        prompt: '% ',
+        input: process.stdin,
+        output: process.stdout,
+        eval: processReplInput,
+        writer: myWriter,
+    });
+    const repl = new Repl(server);
```

Comment:
This looks good. Over time will will move most of this other code into methods on the class, but we don't have to do it just yet.